apps/music/elements/music-search-box.js

```diff
@@ -200,14 +159,4 @@ function debounce(fn, ms) {
   };
 }
 
-try {
```

Comment:
Keep this try/catch since we use it in all our custom elements. Will help catch exceptions in case WC is unsupported in an environment (e.g. running on desktop).